tests/frontend/onnx/test_build_ir.py

```diff
@@ -59,3 +62,108 @@ def test_visit_value_info():
 
     # check it is returned the same reference
     assert t1 is t2
+
+
+def test_visit_node_unknown_op_name():
+    """
+    Test for unknown expected onnx op
+    """
+    node_attributes = {
+        "name": "dummy_name",
+        "op_type": "dummy_op",
+    }
+
+    node = helper.make_node(
+        **node_attributes, inputs=["input1", "input2"], outputs=["output"]
+    )
+    ctx = OnnxXdslMapping()
+    with pytest.raises(ValueError, match="Unknown ONNX op name dummy_op"):
+        visit_node(node=node, ctx=ctx)
+
+
+def test_visit_node_add():
+    # initialize context
+    ctx = OnnxXdslMapping()
+
+    # create graph composed only of one Sub operation
+    graph, sub_node = _create_graph_binary_op("Add", "add_graph")
+
+    # update context
+    _update_context(graph, ctx)
+
+    # expected output before visinting the op node
+    expected_output_pre = """{'input1': <BlockArgument[tensor<0x0xf32>] index: 0, uses: 0>, 'input2': <BlockArgument[tensor<0x0xf32>] index: 1, uses: 0>}"""
+
+    assert str(ctx.value_by_name) == expected_output_pre
+
+    # visit node
+    visit_node(sub_node, ctx)
```

Comment:
could you please add a test for the output of this function?

Reply:
@superlopuh , visit_node returns nothing as output, it just update the content of ctx.
Is there something else I could test?

Reply:
I added the Operation as function output and compare the name of the operation in the test.

Reply:
(Done)